virttest/env_process.py

```diff
@@ -575,6 +576,16 @@ def preprocess(test, params, env):
             except path.CmdNotFoundError, msg:
                 raise exceptions.TestSkipError(msg.message)
 
+    # enable network proxies setting in urllib2
+    if params.get("network_proxies"):
+        proxies = {}
+        for proxy in re.split("[\,\;]\s*", params["network_proxies"]):
```

Comment:
@xutian I'm wondering what the `\` in front of `,` and `;` does. Wouldn't `r"[,;]\s*"` suffice? Also please use the `r` prefix when specifying regexp...

Btw about the `\s*`, it only removes empty spaces after the delimiter. How about `r"\s*;\s*|\s*,\s*"`? It leaves only beginning and tailing whitespaces and those should be removed by cartesian. Anyway it's minor as users will cope with whatever is out there...